mafia.py

```diff
@@ -1,110 +1,171 @@
 from sys import argv
 from random import randrange, shuffle
+from typing import Callable, Dict, List, Text, Tuple
```

Comment:
I personally don't see any reason not to use it.
Sure we could use docstrings too but for more rewarding info imo.
Using too much doc is just gonna get our code cluttered and slow to read by others.
But still, I could remove them after after I'm done with those functions, if u insist.

Reply:
This kind of variable assignment is reminding **_Rust_** for me and I don't want the code be that much frightening to look at it, for instance it may be obvious for reader to understand that `is_...` may return `True` or `False`.